pkg/scheduler/flavorassigner/flavorassigner_test.go

```diff
@@ -823,14 +819,16 @@ func TestAssignFlavors(t *testing.T) {
 						Resource(corev1.ResourceCPU, "1").
 						FlavorQuotas,
 				).Cohort("test-cohort").ClusterQueue,
-
-			cohortResources: &cohortResources{
-				requestableResources: resources.FlavorResourceQuantities{
-					{Flavor: "one", Resource: corev1.ResourceCPU}: 10_000,
-				},
-				usage: resources.FlavorResourceQuantities{
-					{Flavor: "one", Resource: corev1.ResourceCPU}: 9_000,
-				},
+			secondaryClusterQueue: utiltesting.MakeClusterQueue("test-secondary-clusterqueue").
+				ResourceGroup(
+					utiltesting.MakeFlavorQuotas("one").
+						ResourceQuotaWrapper(corev1.ResourceCPU).NominalQuota("10").LendingLimit("0").Append().
```

Comment:
Should this lending limit be present?

Reply:
I think so to achieve this status, otherwise it will borrow
"insufficient unused quota in cohort for cpu in flavor one, 1 more needed"